src/stirling/source_connectors/socket_tracer/protocols/mongodb/parse.cc

```diff
@@ -67,16 +67,19 @@ ParseState ParseFrame(message_type_t type, std::string_view* buf, Frame* frame,
   // Parser will ignore Op Codes that have been deprecated/removed from version 5.0 onwards.
   if (!(frame_type == Type::kOPMsg || frame_type == Type::kOPCompressed ||
         frame_type == Type::kReserved)) {
```

Comment:
Can we remove `Type::kReserved` and `Type::kOPCompressed` from this if statement to avoid adding the `else if` branch added in this PR?

I ran the updated unit test and it seems to pass still.

Reply:
Will modify the comment as well